### PR TITLE
Remove python2 compat

### PR DESCRIPTION
**Pull request by xi**
This is an extended version of #274 that removes all python2 compatibility code.

I have not much experience with metaclasses, so someone smarter than me should check if this requires a migration or something.

### CODE REVIEW

File: parler/admin.py [hunk 1]
```diff
@@ -39,7 +39,6 @@ class ProjectAdmin(TranslatableAdmin):
 from __future__ import unicode_literals
```
Comment (jrief) [inline parler/admin.py:39]: can be removed after dropping Python-2

File: parler/managers.py [hunk 2]
```diff
@@ -56,11 +55,11 @@ def _extract_model_params(self, defaults, **kwargs):
                     pass
 
         if django.VERSION < (2, 2):
-            lookup, params = super(TranslatableQuerySet, self)._extract_model_params(defaults, **kwargs)
```
Comment (jrief) [inline parler/managers.py:59]: Django-2.1 and below reached EOL. Shouldn't we drop this altogether?

File: setup.py [hunk 3]
```diff
@@ -38,9 +38,8 @@ def find_version(*parts):
     version=find_version('parler', '__init__.py'),
     license='Apache 2.0',
 
-    install_requires=['six'],
     requires=[
-        'Django (>=1.7)',
+        'Django (>=2.0)',
```
Comment (jrief) [inline setup.py:42]: >=2.2

File: setup.py [hunk 4]
```diff
@@ -66,15 +65,15 @@ def find_version(*parts):
...
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
         'Framework :: Django',
-        'Framework :: Django :: 1.11',
         'Framework :: Django :: 2.0',
         'Framework :: Django :: 2.1',
```
Comment (jrief) [inline setup.py:73]: remove

File: tox.ini [hunk 5]
```diff
@@ -1,19 +1,19 @@
 [tox]
 envlist=
-    py27-django{111},
-    py36-django{111,20,21,22,30},
-    py37-django{20,21,22,30},
+    py36-django{20,21,22,30,31},
```
Comment (jrief) [inline tox.ini:3]: drop Django-2.0,2.1

File: tox.ini [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
+    py38-django{20,21,22,30,31},
     # py33-django-dev,
     coverage,
     docs,
 
 [testenv]
 deps = django-polymorphic
-    django111: Django >= 1.11,<1.12
     django20: Django >= 2.0,<2.1
     django21: Django >= 2.1,<2.2
```
Comment (jrief) [inline tox.ini:13]: drop

**Review by jrief — CHANGES_REQUESTED** [review]
Thanks you very much for this huge effort.
I'm going to merge and release it as version 2.2 ASAP.
Sorry for the delay.

File: parler/managers.py (hunk 2 above)
Comment (xi) [reply parler/managers.py:59]: Sure!

**Comment by xi** [comment]
Thanks for the feedback! I made the requested changes.

**Review by jrief — APPROVED** [review]

**Pull request merged**